### PR TITLE
Relocate 'When things go wrong' and make it fail with a _config typo

### LINKED ISSUE #204: Move "how to diagnose jekyll errors" to the top of the includes lesson
During the lesson some users made typos `{% includes filename %}` instead of `{% include filename %}` and we ended up having to troubleshoot failures just before we explain how to get the cause of the error.

We could solve this by moving the section to the top of the lesson

### PR DESCRIPTION
**Pull request by unode**
Fixes #204

This example is a little less contrived than the previous.
It can happen to learners as soon as we start tinkering with `_config.yml` so has been relocated to the "Starting with Jekyll" lesson.

### CODE REVIEW

File: _episodes/starting-jekyll.md [hunk 1]
```diff
@@ -165,6 +167,84 @@ Let's make use of global parameters in our pages.
...
 
+## When things go wrong
+
+So far we have seen how to successfully use Jekyll to produce a website.
+There are however some situations where Jekyll may fail to do so
+either due to a typo or missing information.
+
+> ## Exercise: Troubleshooting Jekyll
+>
+> This exercise will help you recognise how common mistakes look like
```
Comment (tobyhodges) [inline _episodes/starting-jekyll.md]:
```suggestion
> This exercise will help you recognise what common mistakes look like
```

File: _episodes/starting-jekyll.md [hunk 2]
```diff
@@ -165,6 +167,84 @@ Let's make use of global parameters in our pages.
...
+If you were keeping an eye on the GitHub repository page until now, you may have noticed 
+a yellow circle visible when the website is still being processed and a green check mark (✓) when successful.
+You may have also noticed that in the same location there is now a red cross next to the commit message (❌).
+
+This indicates that something went wrong with our Jekyll build process.
+
+![Jekyll pending/successful build](../fig/includes_fail_pending_successful.png)
+
+You may also find an email from GitHub in your inbox with details about the error.
+But lets look at our repository again.
```
Comment (tobyhodges) [inline _episodes/starting-jekyll.md]:
```suggestion
But let's look at our repository again.
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Excellent work @unode only a couple typo fixes suggested.

File: _episodes/starting-jekyll.md (hunk 2 above)
Comment (unode) [reply _episodes/starting-jekyll.md]: I do this one all the time. Thanks Toby!

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**